test/api/assignments_test.rb

```diff
@@ -126,12 +130,13 @@ def test_notify_team_members_about_submission
   end
 
   def test_api_rejects_duplicates
-    Attempt.new(alice, Iteration.new({ code: 'THE CODE', path: 'ruby/one/code.rb'}, 
+    Attempt.new(alice, Iteration.new({'THE CODE' => 'ruby/one/code.rb'}, 
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.